src/app/results-list/results-list.component.ts

```diff
@@ -0,0 +1,10 @@
+import {Component, Input} from '@angular/core';
+
+@Component({
+  selector: 'app-results-list',
+  templateUrl: './results-list.component.html',
+  styleUrls: ['./results-list.component.css']
+})
+export class ResultsListComponent {
+  @Input() cocktails: Array<Object> = [];
```

Comment:
Why we use "@Input" ?

Reply:
Because this is child component and we get these cocktails from parent

Reply:
ok, I got